pkgs/applications/science/math/wolfram/default.nix

```diff
@@ -58,14 +58,23 @@ let versions = callPackage ./versions.nix { };
 
 in
 
-callPackage ./generic.nix {
+callPackage ./generic.nix rec {
+
   inherit cudaSupport cudaPackages;
+
   inherit (found-version) version lang;
+
   src = if source == null then found-version.src else source;
-  name = ("mathematica"
+
+  basename = if lib.versionAtLeast found-version.version "14.1.0"
```

Comment:
This causes evaluation to error because `basename` is not a valid argument to `generic.nix`.